install/kubernetes/cilium/templates/cilium-envoy/daemonset.yaml

```diff
@@ -0,0 +1,254 @@
+{{- if and .Values.envoy.enabled (not .Values.preflight.enabled) }}
+
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: cilium-envoy
+  namespace: {{ .Release.Namespace }}
+  labels:
+    k8s-app: cilium-envoy
+    app.kubernetes.io/part-of: cilium
+    app.kubernetes.io/name: cilium-envoy
+    name: cilium-envoy
+spec:
+  selector:
+    matchLabels:
+      k8s-app: cilium-envoy
+  {{- with .Values.envoy.updateStrategy }}
+  updateStrategy:
+    {{- toYaml . | trim | nindent 4 }}
+  {{- end }}
+  template:
+    metadata:
+      annotations:
+        {{- if and .Values.envoy.prometheus.enabled (not .Values.envoy.prometheus.serviceMonitor.enabled) }}
+        prometheus.io/port: "{{ .Values.envoy.prometheus.port }}"
+        prometheus.io/scrape: "true"
+        {{- end }}
+        {{- if .Values.envoy.rollOutPods }}
+        # ensure pods roll when configmap updates
+        cilium.io/cilium-envoy-configmap-checksum: {{ include (print $.Template.BasePath "/cilium-envoy/configmap.yaml") . | sha256sum | quote }}
+        {{- end }}
+        {{- if not .Values.envoy.securityContext.privileged }}
+        # Set app AppArmor's profile to "unconfined". The value of this annotation
+        # can be modified as long users know which profiles they have available
+        # in AppArmor.
+        container.apparmor.security.beta.kubernetes.io/cilium-envoy: "unconfined"
+        {{- end }}
+        {{- with .Values.envoy.podAnnotations }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
+      labels:
+        k8s-app: cilium-envoy
+        name: cilium-envoy
+        app.kubernetes.io/name: cilium-envoy
+        app.kubernetes.io/part-of: cilium
+        {{- with .Values.envoy.podLabels }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      {{- with .Values.envoy.podSecurityContext }}
+      securityContext:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      containers:
+      - name: cilium-envoy
+        image: {{ include "cilium.image" .Values.envoy.image | quote }}
+        imagePullPolicy: {{ .Values.envoy.image.pullPolicy }}
+        command:
+        - /usr/bin/cilium-envoy
+        args:
+        - '-c /var/run/cilium/envoy/bootstrap-config.json'
+        - '--base-id 0'
+        {{- if and (hasKey .Values.debug "verbose") ( has "envoy" .Values.debug.verbose) }}
```

Comment:
@sayboras 

thanks. i assumed that this is an array and not a comma-separated list :man_facepalming: 

anyway - fixed it. it can handle commaseparated values in the `debug.verbose` parameter.